lib/rules/structure/neutral.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @file make sure specification use netural words.
+ */
+const self = {
+    name: 'structure.neutral'
+};
+const blocklist = [
+    'master', 'masters', 'mastered',
+    'slave', 'slaves',
+    'grandfather', 'sanity', 'dummy',
+    'whitelist', 'whitelists', 'blacklist', 'blacklists',
+    'he', 'him', 'his',
+    'she', 'her', 'hers'
+];
+
+exports.name = self.name;
+
+exports.check = function (sr, done) {
+    const blocklistReg = new RegExp('\\b' + blocklist.join('\\b|\\b') + '\\b', 'i');
+    let unneutralList = [];
+    Array.prototype.forEach.call(sr.jsDocument.body.children, child => {
```

Comment:
You can process `sr.jsDocument.body` directly instead of its children. This will save you from iterating through each of them with `forEach`